src/lsdb/nested/core.py

```diff
@@ -435,6 +435,11 @@ def compute(self, **kwargs):
         """Compute this Dask collection, returning the underlying dataframe or series."""
         return npd.NestedFrame(super().compute(**kwargs))
 
+    @property
+    def exploded_columns(self) -> list[str]:
```

Comment:
I feel like I want a unit test for this method. My gut is that this would give `["id", "ra", "dec", "sources", "sources.mjd", "sources.mag"]`, but I'm not so sure about the `"sources"` entry.